mala/interfaces/ase_calculator_uncertainty.py

```diff
@@ -0,0 +1,157 @@
+"""ASE calculator for MALA predictions."""
```

Comment:
This file probably started as a copy of `mala/interfaces/ase_calculator.py`, as such many docs/comments may or may not apply, e.g. `"ASE calculator for MALA predictions."`.

Reply:
That's correct, I will have to check once development finishes that I do not miss any comments or docstrings.